src/base/preferences.h

```diff
@@ -174,6 +174,8 @@ class Preferences: public QObject
     void setWebUiLocalAuthEnabled(bool enabled);
     QString getServerDomains() const;
     void setServerDomains(const QString &str);
+    QString getWebUiAddress() const;
+    void setWebUiAddress(const QString& addr);
```

Comment:
Incorrect coding style.

Reply:
I'm always reluctant to point that out because there is no rule for it.